.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -158,6 +158,7 @@ Release Week Checklist:
 - [ ] **Check for presence of jdk8u aarch32 GA tag and mirror it** [Upstream Git repo](https://github.com/openjdk/aarch32-port-jdk8u) - [Mirror job](https://ci.adoptium.net/view/git-mirrors/job/git-mirrors/job/adoptium/job/git-skara-aarch32-jdk8u/)
 - [ ] **Do all of the above for the jdk8u/aarch32 build: Ensure to specify overridePublishName param**
 - [ ] **Archive/upload all TCK results**
+- [ ] **Archive/upload all AQA results** [Publish AQA test results](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md)
```

Comment:
> Too bad we can not hyperlink directly to an anchor at that page

Yeah that's a fair comment. We could maybe say something like this to make it a little easier but I'm happy to approve regardless :-)
```suggestion
- [ ] **Archive/upload all AQA results** Search for `Publish AQA test results` in [RELEASING.md](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md) for the process.
```